src/screens/TopicPageScreen/index.js

```diff
@@ -396,10 +401,14 @@ const TopicPageScreen = (props) => {
     />
   );
 
-  if (isInitialLoading) return null;
   return (
-    <SafeAreaProvider forceInset={{top: 'always'}} style={styles.parentContainer}>
+    <View>
       <NavHeader
+        initialData={{
+          channelPicutre: params.channelPicture
+        }}
+        isLoading={isInitialLoading}
+        opacityImage={opacityImage}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes here seem to be related to the loading state of the component. The `NavHeader` component now receives an `isLoading` prop and a new `initialData` prop. It's important to ensure that these props are being handled correctly within the `NavHeader` component itself. 

```diff
+ return (
+   <View>
+     <NavHeader
+       initialData={{
+         channelPicutre: params.channelPicture
+       }}
+       isLoading={isInitialLoading}
+       opacityImage={opacityImage}
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->